package.json

```diff
@@ -23,18 +23,18 @@
 	],
 	"scripts": {
 		"build": "tsup",
-		"create:test": "node script/create-test-e2e.js",
+		"test:create": "node script/create-test-e2e.js",
 		"format": "prettier \"**/*\" --ignore-unknown",
 		"format:write": "pnpm format --write",
 		"initialize": "tsx ./src/bin/index.js --mode initialize",
-		"initialize:test": "node script/initialize-test-e2e.js",
+		"test:initialize": "node script/initialize-test-e2e.js",
 		"lint": "eslint . .*js --max-warnings 0",
 		"lint:knip": "knip",
 		"lint:md": "markdownlint \"**/*.md\" \".github/**/*.md\" --rules sentences-per-line",
 		"lint:package-json": "npmPkgJsonLint .",
 		"lint:packages": "pnpm dedupe --check",
 		"lint:spelling": "cspell \"**\" \".github/**/*\"",
-		"migrate:test": "node script/migrate-test-e2e.js",
+		"test:migrate": "node script/migrate-test-e2e.js",
```

Comment:
You'll need to update the GitHub workflows too:

https://github.com/JoshuaKGoldberg/create-typescript-app/blob/edd4d8fdccc44ae77dd8a6c15b6d5cf26024f80a/.github/workflows/test-migrate.yml#L8